configure.ac

```
@@ -1207,6 +1207,16 @@ SQUID_CHECK_LIB_WORKS(ldap,[
   SQUID_STATE_ROLLBACK(squid_ldap_state)
 ])
 
+SQUID_AUTO_LIB(sasl,[LIBSASL],[libsasl2])
```

Parameter two is the library name/description for use in the `./configure --help` text.
Parameter 3 must be the same variable passed to `PKG_CHECK_MODULES()` parameter 1.
```suggestion
SQUID_AUTO_LIB(sasl,[Cyrus SASL],[LIBSASL])
```